### PR TITLE
Docs: migration guides order

### LINKED ISSUE #9550: Migration guides should be sorted by latest version first
## 📝 Provide a description of requested docs changes

[Currently the sidebar in our builds docs](https://ckeditor.com/docs/ckeditor5/27.1.0/builds/index.html) shows:

Typically you care only about x last releases and versions from year ago bother proportionally smaller audience.

Also having the latest version first is the most natural to me (and I believe to most users). E.g. that's the way [GH orders our releases](https://github.com/ckeditor/ckeditor5/releases) same thing goes for our changelog, etc.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: Reversing migration guides order in TOC. Closes #9550.

Docs: Extending the build landing page just a tiny little bit. Partially addresses cksource/docs#368

### CODE REVIEW

File: docs/builds/guides/migration/migration-to-27.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 category: builds-migration
 menu-title: Migration to v27.x
-order: 10
```
Comment (Reinmar) [inline docs/builds/guides/migration/migration-to-27.md:4]: 97?

File: docs/builds/index.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ CKEditor 5 Builds are the fastest and easiest way to use CKEditor 5 in your appl
 	Use the <span class="navigation-hint_desktop">**navigation tree on the left**</span><span class="navigation-hint_mobile">**main menu button in the upper-left corner**</span> to navigate through CKEditor 5 Builds documentation.
 </info-box>
 
+Learn more about the {@link builds/guides/overview available pre-configured builds} or {@link builds/guides/development/custom-builds creating custom builds} and updating them on the go with the {@link builds/guides/development/dll-builds DLL webpack} solution.
+
+Get to know more about the {@link builds/guides/development/plugins plugins development} and the supported {@link builds/guides/frameworks/overview frameworks integrations}.
```
Comment (AnnaTomanek) [inline docs/builds/index.md:20]:
```suggestion
Get to know more about the {@link builds/guides/development/plugins plugin development} and the supported {@link builds/guides/frameworks/overview integrations with popular JavaScript frameworks}.
```

File: docs/builds/index.md [hunk 3]
```diff
@@ -15,6 +15,12 @@ CKEditor 5 Builds are the fastest and easiest way to use CKEditor 5 in your appl
...
+
+Refer to the {@link builds/guides/migration/migration-from-ckeditor-4 CKEditor 4 migration guide} or other migration guides if you are updating your integration.
```
Comment (AnnaTomanek) [inline docs/builds/index.md:22]:
```suggestion
Refer to the {@link builds/guides/migration/migration-from-ckeditor-4 CKEditor 4 migration guide} or other migration guides if you are updating your CKEditor 5 installation.
```

File: docs/builds/guides/migration/migration-to-27.md (hunk 1 above)
Comment (AnnaTomanek) [inline docs/builds/guides/migration/migration-to-27.md:4]:
```suggestion
order: 97
```

Comment (godai78) [reply docs/builds/guides/migration/migration-to-27.md:4]: Yes. Then v28 gonna be 96 and then so on and we do not need to rewrite it for like 100 releases, which is about a decade.

Comment (godai78) [reply docs/builds/guides/migration/migration-to-27.md:4]: Covered in a commit.

**Review by AnnaTomanek — APPROVED** [review]

**Pull request merged**